test/unit/lib/monocle_test.js

```diff
@@ -42,6 +42,14 @@ describe('Monocle API Client', function() {
                     this.http.request.calledWith(method.toUpperCase(), '/my-base/foo').should.be.true;
                 });
 
+                it('prepends host if provided', function() {
```

Comment:
in node_tests there is a test that checks whether errors are rejected properly.... if we don't set a host or set a non existing host for example this request will return an error and we are stubbing this with this test